run/AM4_run_script.sh

```diff
@@ -29,9 +29,9 @@ executable=/path/to/executable/fms_cm4p12_warsaw.x
 
 
 ## Run parameters
-#total_npes is the number of cores to run on, omp_threads is the number of 
+#total_npes is the number of cores to run on, omp_threads is the number of
 # openMP threads
-total_npes=216
+total_npes=432
 omp_threads=1
```

Comment:
The current namelist has 2 atmos_threads, so either the namelist needs to be updated, or this line of the script needs to be updated.